web/satellite/src/utils/metaContentExtractor.ts

```diff
@@ -0,0 +1,14 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+export class MetaRepository {
```

Comment:
we should find better name for this file and for this class. Let's think what intention this class has